src/plugins/intel_gpu/include/intel_gpu/graph/serialization/tensor_serializer.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit: same as above

Reply:
I updated the copyright year here too. Thank you.